src/main/java/com/code_intelligence/jazzer/junit/AgentConfigurator.java

```diff
@@ -52,6 +52,8 @@ static void forFuzzing(ExtensionContext executionRequest) {
     String filter = String.join(File.pathSeparator, instrumentationFilter.split(","));
     System.setProperty("jazzer.custom_hook_includes", filter);
     System.setProperty("jazzer.instrumentation_includes", filter);
+    System.setProperty("jazzer.instrumentation_excludes",
+        String.join(File.pathSeparator, "org.apache.maven.**, org.gradle.**"));
```

Comment:
These classes *should* not be matched by the default instrumentation filter we compute in line 51. Do the projects with which you are experiencing issues have `jazzer.instrument` set?

Reply:
I don't think so.